MSAL/test/app/MSALStressTestHelper.h

```diff
@@ -0,0 +1,41 @@
+//
+//  MSALStressTestHelper.h
+//  MSAL
+//
+//  Created by Olga Dalton on 4/24/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
```

Comment:
Make sure you update the header, you can prob copy and paste off of other classes